Rejected-or-Rolled-Back-Features.md

```diff
@@ -1,28 +1,27 @@
-In addition to list below make sure to check [list of closed issues and pull requests labeled as `wontfix`](https://github.com/runelite/runelite/issues?utf8=%E2%9C%93&q=label%3Awontfix).
+In addition to list below make sure to check [list of closed issues and pull requests labeled as `wontfix`](https://github.com/runelite/runelite/issues?utf8=%E2%9C%93&q=label%3Awontfix) and [Jagex's current guidelines for third-party clients](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1).
 
 Jagex has requested removal of certain features, and current discussion of feature requests on Discord and GitHub concluded that RuneLite's stance on these features are as follows:
 
 #### Removed in the past
 * AoE Plugin: Removed at Jagex's request (see [1.2.13 blog](https://runelite.net/blog/show/2018-01-25-1.2.13-Release))
 * In-game Zulrah Helper: Removed at Jagex's request (see [1.2.13 blog](https://runelite.net/blog/show/2018-01-25-1.2.13-Release), relevant discussion in [#1386](https://github.com/runelite/runelite/issues/1386))
 * Volcanic Mine Helper: Removed at Jagex's request (see [1.2.13 blog](https://runelite.net/blog/show/2018-01-25-1.2.13-Release))
-* Demonic Gorilla plugin - removed at Jagex's request (https://secure.runescape.com/m=news/a-message-about-unofficial-clients?oldschool=1)
+* Demonic Gorilla plugin - [removed at Jagex's request](https://secure.runescape.com/m=news/a-message-about-unofficial-clients?oldschool=1)
 * Barbarian Assault Plugin: Left-click calling removed (see [#859](https://github.com/runelite/runelite/pull/859))
 * Fight Cave Plugin: Plugin trivialises the fight against Jad (see [#6724](https://github.com/runelite/runelite/pull/6724))
-* Cerberus plugin - [see Jagex statement](https://secure.runescape.com/m=news/another-message-about-unofficial-clients?oldschool=1)
-* Prayer reorder plugin - [see Jagex statement](https://secure.runescape.com/m=news/another-message-about-unofficial-clients?oldschool=1)
+* Cerberus plugin - [see Jagex statement](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1)
```

Comment:
* Cerberus plugin: [see Jagex statement](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1)
change - to : for consistency